haiku-apps/wordclock/wordclock-1.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An imprecise clock replicant"
+DESCRIPTION="WordClock is imprecise clock replicant and this show clock with word"
```

Comment:
That's not very nicely phrased. How about:
"WordClock is an imprecise clock that displays the time in words instead of digits. It can be used as a Replicant as well."